pageserve/templates/events.html

```diff
@@ -36,4 +36,28 @@ <h1>{% block title %}Sub-Events{% endblock title %}</h1>
 {% endfor %}
 
 <a href='/v1/'>See all posts</a>
-{% endblock content%}
\ No newline at end of file
+
+<h2>Create New Event</h2>
+<!-- TODO: add correct endpoint -->
+<form action="/v1/add_event" method="post">
+    <div>
+        <label for="name">Event Name:</label>
+        <input type="text" id="name" name="event_name">
+    </div>
+    <div>
+        <label for="description">Description:</label>
+        <textarea type="text" rows = "5" cols = "60" id="description" name="description">
+          Enter details here.
+        </textarea>
+    </div>
+    <div>
+        <label for="event_time">Event Time:</label>
+        <input type="datetime-local" id="event_time" name="event_time">
+    </div>
+    <input type="hidden" id="author" name="author_id" value="{{user}}">
```

Comment:
What might be better than a hidden field is not sending this from the front end and instead having `add_event()` handle this. In `add_event()`, you could make it get the user object (I'm working on writing a user authentication helper function to make this one call, for now you could just use a placeholder). Then if the returned user exists and is an organizer, you can send it along to the request with `data={**request.form, author_id=user['user_id']}`. Same goes for the post form, but you wouldn't have to check for `is_organizer` in `add_event()`.

Reply:
+1 for checking this with authentication server-side.

Reply:
Good idea, done in 3b470a5